chromium_src/components/policy/resources/policy_templates.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import override_utils
+import json
+
+
+@override_utils.override_function(globals())
+def _LoadPolicies(orig_func):
+    policies = orig_func()
+
+    # `policies` will have the following notable keys:
+    #
+    # "policy_definitions"
+    # there will be one "group" for every folder found under
+    # `src/components/policy/resources/templates/policy_definitions`
+    # Chromium considers the folder name the group name for the policy.
+    # Brave uses the group name "BraveSoftware". The child element for the
+    # group is the policy itself (those are the yaml files in the folder).
+    #
+    # Brave specific entries will get copied into place by the hooks that run
+    # via DEPS. We copy the files from:
+    # `src/brave/components/policy/resources/templates/policy_definitions/BraveSoftware` # pylint: disable=line-too-long
+    # to:
+    # `src/components/policy/resources/templates/policy_definitions`
+    #
+    #
+    # "policies"
+    # This has the contents of:
+    # `src/components/policy/resources/templates/policies.yaml`
+    # This is where we need to inject the Brave specific names. The policies
+    # themselves are already defined (under `policy_definitions`), we just need
+    # to add a mapping for ID (integer; unique) and name (matches name under
+    # `policy_definitions`).
+    #
+    #
+    # There are some other fields which are not used by this script:
+    # - "common_schemas"
+    # - "legacy_device_policy_proto_map"
+    # - "manual_device_policy_proto_map"
+    # - "risk_tag_definitions"
+
+    policy_yaml = policies['policies']
+    policy_section = policy_yaml['policies']
+
+    # get the highest ID in the file
+    highest_number = 0
+    for key, _ in policy_section.items():
+        if int(key) > highest_number:
+            highest_number = int(key)
+
+    # append our entries to the ones from policies.yaml
+    # TODO(bsclifton): we can create this array dynamically by reading the file
+    # names from:
+    # `src/brave/components/policy/resources/templates/policy_definitions/BraveSoftware` # pylint: disable=line-too-long
+    brave_policies = [
+        'TorDisabled', 'BraveRewardsDisabled', 'BraveWalletDisabled',
+        'BraveVPNDisabled', 'BraveAIChatEnabled', 'BraveSyncUrl',
+        'BraveShieldsDisabledForUrls', 'BraveShieldsEnabledForUrls'
+    ]
```

Comment:
if you move the copying of the files into this script, then you don't need to list those here manually.

Reply:
Fixed. No need to examine the files during copy (although that was great to combine the two). The other field policy_definitions had all the items all along 😄 Updated!